src/WebJobs.Extensions.DurableTask/Correlation/TraceActivityConstants.cs

```diff
@@ -10,8 +12,12 @@ internal class TraceActivityConstants
         public const string Activity = "activity";
         public const string Event = "event";
         public const string Timer = "timer";
+        public const string Entity = "entity";
 
         public const string CreateOrchestration = "create_orchestration";
         public const string OrchestrationEvent = "orchestration_event";
+
+        public const string CallEntity = "CallEntity";
+        public const string SignalEntity = "SignalEntity";
```

Comment:
Looking at other constants in this file, it seems we're using snake_case for the values. For example `create_orchestration` and `orchestration_event`. Should we be doing the same for these new constants you're defining?